src/Service.php

```diff
@@ -136,9 +110,68 @@ public function getPermissions(Concrete $object, array $visitedIds = []): array
      * @param string $resource
      * @return bool
      */
-    public function isAllowed(Concrete $object, $resource): bool {
+    public function isAllowed(Concrete $object, $resource): bool
+    {
         $permissions = $this->getPermissions($object);
         return $permissions[$resource] == self::ALLOW;
     }
 
+    /**
+     * @param Concrete|Objectbrick\Data\AbstractData $object
+     * @param ClassDefinition\Data $fieldDefinition
+     * @param bool $ignoreObjectBrickFields
+     * @return array
+     */
+    protected function getPermissionsByFieldDefinition(
+        AbstractModel $object,
+        ClassDefinition\Data $fieldDefinition,
+        bool $ignoreObjectBrickFields = false
```

Comment:
is that param really needed?

Reply:
I guess it's optional. I added it since the original logic didn't contain a condition for object bricks within object bricks, but I suppose that situation will never arise.

Reply:
saw that one too late :-)